package.json

```diff
@@ -86,6 +86,7 @@
     "electron-updater": "5.0.1",
     "katex": "0.16.2",
     "keytar": "7.9.0",
+    "mitt": "^3.0.1",
```

Comment:
Have you tried to move this to devDependencies?

Reply:
My bad. It is in devDependencies now.